source/distributed/shared_tria.cc

```diff
@@ -134,6 +134,16 @@ namespace parallel
       this->update_number_cache ();
     }
 
+    template <int dim, int spacedim>
+    void
+    Triangulation<dim, spacedim>::
+    copy_triangulation (const dealii::Triangulation<dim, spacedim> &old_tria)
+    {
+      dealii::Triangulation<dim,spacedim>::copy_triangulation (old_tria);
```

Comment:
i don't think one should call it in `copy_triangulation`, as you may as well keep both triangulations at the same time.